src/registrar/views/domain.py

```diff
@@ -335,6 +336,8 @@ def _make_invitation(self, email_address):
             )
         else:
             # created a new invitation in the database, so send an email
+            domaininfo = DomainInformation.objects.filter(domain=self.object)
+            full_name = domaininfo.first().creator
```

Comment:
```suggestion
           full_name = f"{domaininfo.first().creator.first_name} {domaininfo.first().creator.last_name}"
```
This addresses @zandercymatics and @Katherine-Osos's comments, but check that it passes linter (line seems long)

Reply:
Approved from me after this change